### DISCUSSION TITLE
Disabling Sourcery.ai app integration

### DISCUSSION THREAD

**Discussion by youtux** [pytest-dev/pytest-bdd#701] [General]
@pytest-dev/core can someone of you disable the Sourcery.ai integration with this repo, `pytest-bdd`?
I'm trying to, but it says that this action can only be performed by an organization owner, which I'm not.

https://github.com/apps/sourcery-ai/installations/43933416

<img width="1190" alt="Screenshot 2024-09-21 at 11 35 07" src="https://github.com/user-attachments/assets/654d477e-87a9-4761-9559-206e274eabe0">

**Comment by nicoddemus** [comment]
This repository was the only one using the integration, so I uninstalled it from pytest-dev. 👍

**Reply by youtux** [reply]
Thank you!
